src/FreeAge/common/type_stats.hpp

```diff
@@ -154,3 +154,11 @@ struct BuildingTypeStats : public ObjectTypeStats {
   /// TODO: doc
   QRect occupancy;
 };
+
+// TODO: move elsewhere
+struct GameData {
```

Comment:
This is **temp** struct to keep all the data (currently the UnitTypeStats and the BuildingTypeStats).